src/toil/batchSystems/lsf.py

```diff
@@ -117,13 +118,12 @@ def getJobExitCode(self, lsfJobID):
                     logger.error("bjobs detected job failed for job: "
                                  "{}".format(job))
                     return 1
-                elif line.find("Started on ") > -1:
+                elif line.find("Started on ") > -1 or line.find(": Started") > -1:
                     started = 1
-
-            if started == 1:
-                logger.debug("bjobs detected job started but not completed: "
-                             "{}".format(job))
-                return None
+                    if started == 1:
```

Comment:
Why the indent here?

Reply:
Because this should only be called if started was found in the line

Reply:
I agree. It looks to me like this indent should not be here. Right now it renders the `if` evaluation useless because you set `started = 1` and then evaluate the state of `started`. I think this is the only place that `started` is used so perhaps could remove that variable assignment / check completely and jump straight to the `logger` + `return None` at this point.

Reply:
The way it was before just didn't make a lot of sense to me, so instead of removing it I put it where it made sense. The only place `if started == 1:` should be used is inside the `elif line.find("Started on ") > -1:` so instead of just removing it I indented it. I can remove or keep.

Reply:
The outer loop runs across all entries in the output log. It has several exits (`return 0`, `return None`, etc). Only if "Started on" was seen AND none of the other conditions are met do we want to issue the error message.

So please undo the indents to this section :-)

Reply:
Thanks for clarifying @mr-c. I was confusing this with `status <RUN>` and thought that this line should be mutually exclusive from all others. Speaking of which - in LSF 10 `Started on` will not be found, but rather `Started 1 Task(s) on ...` so we may need to change this line still.

Reply:
@drjrm3 
```
                elif line.find("Started on ") > -1 or line.find(": Started") > -1:
                    started = 1
```
This should work right?

Reply:
@mr-c - now I'm confused again. Shouldn't we only enter this if the job is in `RUN` state? And isn't it fully expected that this function will be invoked at times when the job is in `RUN` state? In this case, shouldn't we modify this to be in line with the other states to check on all mutually exclusive, exhaustive conditions? I'm almost leaning towards checking the job status initially with `line.split('Status <')[1].split('>')[0]` and then looping over the mutually exclusive cases (including `RUN`, `ZOMBI`, `UNKWN`, etc.). What benefits does it provide to look over all other cases and then mark it as running at the end?